contracts/message-bus/MessageBus.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity >=0.5.0 <0.6.0;
```

Comment:
Will there be separate ticket/PR for unit tests for MessageBus?

Looking at the multi level inheritance, I feel when `ConsensusGateway is MessageBus`, it's going to be huge contract. With test we can know how much gas it is taking and if that's problematic for deployment 🤔 .

Reply:
Added ticket #141, #142, and #143.
I think that the contract code size will be still in deployable range. If it becomes a problem we can solve that.
Also, the contacts using this will follow proxy pattern, so we need to just deploy one master copy of the contract.